bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

```diff
@@ -391,7 +391,7 @@ public class DefaultSystemChannelTypeProvider implements ChannelTypeProvider {
      * Electrical-energy: system wide {@link ChannelType} which shows the electrical energy
      */
     public static final ChannelType SYSTEM_ELECTRICAL_ENERGY = ChannelTypeBuilder
-            .state(SYSTEM_CHANNEL_TYPE_UID_ELECTRICAL_ENERGY, "Electrical Energy", "Number:Energy")
+            .state(SYSTEM_CHANNEL_TYPE_UID_ELECTRIC_ENERGY, "Electrical Energy", "Number:Energy")
```

Comment:
Should the label also be updated accordingly?
```suggestion
            .state(SYSTEM_CHANNEL_TYPE_UID_ELECTRIC_ENERGY, "Electric Energy", "Number:Energy")
```

and description below: